bpf/lib/common.h

```diff
@@ -748,6 +748,16 @@ enum metric_dir {
  */
 #define DSR_IPV4_OPT_TYPE	(IPOPT_COPY | 0x1a)
 
+/* No operation IPv4 option. It has no length.
+ *
+ * Copy = 0 (option is copied to each fragment)
+ * Class = 0 (control option)
+ * Number = 1 (not used according to [1])
+ * 
+ * See: https://www.rfc-editor.org/rfc/rfc791.html
+ */
+#define NOP_IPV4_OPT 0x01
```

Comment:
Note that `bpf/include/linux/ip.h` already gives us `IPOPT_NOOP` (and `IPOPT_END`).